src/mini/template.html

```diff
@@ -29,7 +29,11 @@
       var module;
       if (this._template === undefined) {
         module = Polymer.DomModule.import(this.is);
-        this._template = module && module.querySelector('template');
+        var template = module && module.querySelector('template');
```

Comment:
Isn't it incorrect to do this if `this._template` is explicitly `null`?

Reply:
Correct, this is all skipped when `this._template === undefined`, including when template is explicitly set to null.